regcomp.c

```diff
@@ -23741,6 +23744,36 @@ S_parse_uniprop_string(pTHX_
                     packWARN(WARN_EXPERIMENTAL__UNIPROP_WILDCARDS),
                     "The Unicode property wildcards feature is experimental");
 
+                if (special_property) {
+                    const char * error_msg;
+                    const char * revised_name = name + i;
+                    Size_t revised_name_len = name_len - (i + 1 + escaped);
+                    HV * table;
+
+
+                    /* Currently, the only 'special_property' is name, which we
+                     * lookup in _charnames.pm */
+
+                    table = load_charnames(newSVpvs("placeholder"),
+                                           revised_name, revised_name_len,
+                                           &error_msg);
+                    if (table == NULL) {
+                        sv_catpv(msg, error_msg);
+                        goto append_name_to_msg;
+                    }
+
+                    /* Farm this out to a function just to make the current
+                     * function less unwieldy */
+                    if (handle_names_wildcard(revised_name, revised_name_len,
+                                &prop_definition))
+                    {
+                        return prop_definition;
+                    }
+
+                    goto failed;
+                }
+
+                prop_values = get_prop_values(table_index);
                 /* Now create and compile the wildcard subpattern.  Use /iaa
```

Comment:
Wasn't this going to be `/miaa`?

Reply:
Yes, but this was explaining only the /i and the /aa portions.  But the /aa portion is better explained in the called function, as is the /m.  And the final sentence of the comment (that you didn't highlight turns out to be obsolete)